app/models/miq_server/ntp_management.rb

```diff
@@ -11,7 +11,7 @@ def ntp_reload
 
     # Bust the settings cache allowing this worker to apply any recent changes made by another (UI) worker
     Vmdb::Settings.reload!
-    ntp_settings = get_config("vmdb").config[:ntp]
+    ntp_settings = config[:ntp]
```

Comment:
My issue with the `config` method naming really came up because of this.

I found it plausible for a local variable or attribute to be called `config` and it took me a bit to realize you had renamed `get_config`. I'm more concerned with the implicit `self` target than the explicit target case (which, as you said, makes this more clear).

Reply:
Uhg, jeah. I see your point now. But maybe doing explicit `self.config` would do? Otherwise, I will go and replace it as discussed before.

Reply:
If we use settings as the method name, then maybe the collision is less likely?  Otherwise, I'm fine with self.settings, though rubocop will likely complain.